indico/modules/events/editing/client/js/components/FileManager/util.js

```diff
@@ -0,0 +1,21 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const FileManagerContext = React.createContext(null);
+
+export const filePropTypes = PropTypes.arrayOf(
```

Comment:
@pferreir Wouldn't it be better if we removed the `arrayOf` from this prop declaration and have `filePropTypes` represent just one file object?

Reply:
Yeah, that might make sense.